Gemfile

```diff
@@ -14,8 +14,8 @@ gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
 gem 'coffee-rails', '~> 4.1.0'
 # See https://github.com/rails/execjs#readme for more supported runtimes
-gem 'therubyracer', '>= 0.12.2',  platforms: :ruby
-gem 'libv8', '>= 3.16.14.15' # Min version for Mac OS 10.11, XCode 8.0
+gem 'mini_racer',  platforms: :ruby
```

Comment:
I forgot, did you say that mini_racer didn't actually make a difference for you?

If that's the case, can I recommend this? (which is what we use in a couple of other apps)
```
gem 'therubyracer', '>= 0.12.3',  platforms: :ruby
gem 'libv8', '>= 3.16.14.19' # Min version for Mac OS 10.11, XCode 9.0, Ruby 2.4
```

Reply:
oh, it actually did make a difference - it installs!